src/pages/waypoint/index.tsx

```diff
@@ -1,15 +1,7 @@
-import { ReactElement } from 'react'
-import BaseNewLayout from 'layouts/base-new'
-
-const WaypointHomePage = (): ReactElement => (
-  <div className="g-grid-container">
-    <h1>Welcome to Waypoint</h1>
-    <ul>
-      <li>This page is a work in progress</li>
-    </ul>
-  </div>
-)
-
-WaypointHomePage.layout = BaseNewLayout
-
-export default WaypointHomePage
+import HomePage from 'views/waypoint/home'
+import server from 'views/waypoint/home/server'
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export async function getStaticProps() {
+  return await server.getStaticProps()
+}
+export default HomePage
```

Comment:
Just requesting some extra whitespace here for readability.

```suggestion
import HomePage from 'views/waypoint/home'
import server from 'views/waypoint/home/server'

// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
export async function getStaticProps() {
  return await server.getStaticProps()
}

export default HomePage
```